pvlib/iotools/sodapro.py

```diff
@@ -301,26 +299,17 @@ def parse_cams(fbuf, integrated=False, label=None, map_variables=True):
     obs_period = data['Observation period'].str.split('/')
 
     # Set index as the start observation time (left) and localize to UTC
-    if (label == 'left') | ((label is None) & (time_step != '1M')):
-        data.index = pd.to_datetime(obs_period.str[0], utc=True)
-    # Set index as the stop observation time (right) and localize to UTC
-    # default label for monthly data is 'right' following Pandas' convention
-    elif (label == 'right') | ((label is None) & (time_step == '1M')):
-        data.index = pd.to_datetime(obs_period.str[1], utc=True)
-
-    # For time_steps '1d' and '1M', drop timezone and round to nearest midnight
-    if (time_step == '1d') | (time_step == '1M'):
+    data.index = pd.to_datetime(obs_period.str[0], utc=True)
+
+    # For time_steps '1d' and '1MS' drop timezone and round to nearest midnight
+    if (time_step == '1d') | time_step.endswith('MS'):  # noqa
         data.index = pd.DatetimeIndex(data.index.date)
```

Comment:
@kandersolar Is it desirable that the index becomes a date instead of datetime for frequencies of day and month?

Reply:
What's the distinction?  Does pandas have some kind of `DateIndex`?

Reply:
No they do not. The operation converts the datetimeindex to just an index (where each element is a datetime.date object.

>data.index = pd.DatetimeIndex(data.index.date)

This is however inconvenient as you can't use the convenient pandas datetime operations such as resample etc.